### PR TITLE
update event api to use v1beta1 core api

### PR DESCRIPTION
**Pull request by tpiperatgod**
1. update event API to use v1beta1 core API
2. add `spec.logVerboseLevel` to config the log verbose level in events-handlers
3. remove `DaprIO.Type` since we can get this value from the component's type
4. add events e2e test

- [x] update the images of the event handlers to `v3`

Signed-off-by: laminar <fangtian@kubesphere.io>

### CODE REVIEW

**Comment by tpiperatgod** [comment]
The event handlers' images are now marked as "latest" and I think it is possible to update these images to `v3` after the functions-framework-go is released.

File: controllers/events/eventsource_controller.go [hunk 1]
```diff
@@ -123,6 +130,10 @@ func (r *EventSourceReconciler) createOrUpdateEventSource(ctx context.Context, l
 	// Generate the eventsource function instance with image eventSourceHandlerImage.
 	r.Function = InitFunction(eventSourceHandlerImage)
 
+	if eventSource.Spec.LogVerboseLevel != nil {
+		r.EventSourceConfig.LogLevel = *eventSource.Spec.LogVerboseLevel
```
Comment (benjaminhuo) [inline controllers/events/eventsource_controller.go:134]: Maybe we can use LogLevel instead of LogVerboseLevel

**Comment by benjaminhuo** [comment]
> The event handlers' images are now marked as "latest" and I think it is possible to update these images to `v3` after the functions-framework-go is released.

Means this PR should be merge after functions-framework-go is released?

**Comment by tpiperatgod** [comment]
> > The event handlers' images are now marked as "latest" and I think it is possible to update these images to `v3` after the functions-framework-go is released.
> 
> Means this PR should be merged after functions-framework-go is released?

That would be better.

**Pull request merged**